homeassistant/components/esphome/voice_assistant.py

```diff
@@ -39,7 +39,8 @@
 
 _LOGGER = logging.getLogger(__name__)
 
-UDP_PORT = 0  # Set to 0 to let the OS pick a free random port
+UDP_PORT_MIN = 10500  # Set to 0 to let the OS pick a free random port
+UDP_PORT_MAX = 10520
```

Comment:
Do we have an "official" range of ports we might want to use here?

Reply:
The Wyoming services use 10300 and 10400, so I picked 10500 for this. I'm not sure where an "official" port would be otherwise specified.

Reply:
Not aware of a better logic for selecting the ports, and it would definitely be an exception using more than 20 devices simultaneously. If there is a potential conflict, let me know, but that also shouldn't really matter, since it's looking for an open port. I'll mark this as resolved for now, if there's no objection.

Reply:
Might eventually want this to be configurable somehow.